### ISSUE TITLE
Is there any need for StepLearningScheduler?

### ISSUE THREAD

**Issue by esube** [fchollet/keras#3754]
I have a StepLearningRateScheduler for both automatic (with a monitored metric) and/or manual (with user specified list of epochs) scheduling.

I have added and example of training 28x4 wide resnet on cifar10 and achieved 5.8% error on CIFAR-10 testing set using keras. Is there any need in keras for this scheduler?

If not I will just close the PR and keep the code in my own separate repo.

Thanks
